blazor/pivot-table/data-binding.md

```diff
@@ -547,6 +552,11 @@ The [Blazor Pivot Table](https://www.syncfusion.com/blazor-components/blazor-piv
 
 The [Blazor Pivot Table](https://www.syncfusion.com/blazor-components/blazor-pivot-table) supports **DynamicObject** data source when the model type is unknown. This can be set in [DataSource](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.PivotView.PivotViewDataSourceSettings-1.html#Syncfusion_Blazor_PivotView_PivotViewDataSourceSettings_1_DataSource) property under [PivotViewDataSourceSettings](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.PivotView.PivotViewDataSourceSettings-1.html) class. Moreover, the data source supports all the available features in the component.
 
+To know about **DynamicObject** data binding in Blazor Pivot Table component, you can check on this video.
```

Comment:
To have a quick glance on how to bind data to the Blazor Pivot Table using **DynamicObject**, watch this video: